web/src/components/MemoEditor.tsx

```diff
@@ -440,10 +429,24 @@ const MemoEditor = () => {
           <button className="action-btn">
             <Icon.Code className="icon-img" onClick={handleCodeBlockBtnClick} />
           </button>
-          <button className="action-btn">
-            <Icon.FileText className="icon-img" onClick={handleUploadFileBtnClick} />
+          <div className="action-btn resource-btn" onMouseOver={handleUploadBtnMouseOver} onMouseOut={handleUploadBtnMouseOut}>
```

Comment:
Use CSS to toggle display status. e.g. SearchBar

https://user-images.githubusercontent.com/24653555/204412760-12947f12-9f17-4d87-8dc5-331778d5307f.mp4